thread_pthread_mn.c

```diff
@@ -575,7 +585,12 @@ timer_thread_register_waiting(rb_thread_t *th, int fd, enum thread_sched_waiting
         }
         else {
             VM_ASSERT(fd >= 0);
+#if HAVE_SYS_EVENT_H
+            EV_SET(&ke[num_events], fd, EVFILT_READ, EV_ADD, 0, 0, (void *)th);
```

Comment:
Depending on different usages, you often see `EV_ADD` included with `EV_CLEAR` or `EV_ONESHOT`. Since the MN code automatically handles unregistering events these possibly are not necessary here.